docs/team/jaivigneshvenugopal.adoc

```diff
@@ -0,0 +1,101 @@
+= Jaivignesh Venugopal - Project Portfolio
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+== Project: Codii
+[https://github.com/CS2103AUG2017-T17-B1/main[Codii]] is a desktop address book application specially designed for debt collectors to manage debtors in a simple manner. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
+
+Debt collectors can store information such as the amount owed, debt borrow date and debt cleared date in addition to debtor’s personal information.
+
+Unique features such as an interest calculator help debt collectors manage debts more efficiently.
+
+Codii is evolved from [https://github.com/nus-cs2103-AY1718S1/addressbook-level4[AddressBook - Level 4]] which is a desktop address book application used for teaching Software Engineering principles.
+
+*Code contributed*: [https://github.com/CS2103AUG2017-T17-B1/main/blob/master/collated/main/jaivigneshvenugopal.md[Functional code]] [https://github.com/CS2103AUG2017-T17-B1/main/blob/master/collated/test/jaivigneshvenugopal.md[Test code]]
+
+=== Enhancement Added: Blacklist/Whitelist command
+
+==== External behavior
+
+---
+#Start of Extract [from: User Guide]#
+
+include::../UserGuide.adoc[tag=blacklist]
+
+include::../UserGuide.adoc[tag=whitelist]
+
+#End of Extract#
+
+---
+
+==== Justification (Blacklist)
+
+As a debt collector, there is an important requirement to categorize debtors into different categories. One of these categories include debtors who have been blacklisted. This branding could have been given by the client banks or the debt collection agencies themselves. Nonetheless, the message is clear; these blacklisted debtors must be handled with caution.
+
+Thus, it is a requirement to have access to a list of blacklisted people. That is essentially what the `blacklist` command does. Moreover, working in the blacklist gives exclusive and focused control over the debtors in this list alone. Meaning, when working in the blacklist, users will not be able to access non-blacklisted persons. This is because, the commands work exclusively on blacklisted debtors alone.
```

Comment:
no need for comma after "this is because"

Reply:
Noted